client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/questions/index.tsx

```diff
@@ -14,6 +14,8 @@ export const Questions: FC = () => {
 			</span>
 			<ExternalLink
 				href="https://wordpress.com/help/contact/"
+				icon
```

Comment:
I added icons to the external links in this PR, which was a discussion that happened [here](https://github.com/Automattic/wp-calypso/pull/92387/commits/cd624833a5ff322a65132301c61ba5a05ea466e2).

<img width="1261" alt="Screenshot 2024-07-12 at 12 54 42" src="https://github.com/user-attachments/assets/ad380fa0-15c3-49ba-9370-f90d6c1596ad">